sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/keyvault_secret_properties.hpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Keyvault Secret Attributes definition
+ */
+
+#pragma once
+
+#include <azure/core/datetime.hpp>
+#include <azure/core/nullable.hpp>
+
+namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
+
+  /**
+   * @brief The Secret attributes managed by the KeyVault service.
+   *
+   */
+  struct KeyvaultSecretProperties final
```

Comment:
You're missing `Version` (important) and `VaultUri` (or `VaultUrl` if C++ uses "URL" over "URI", which is true for everyone but .NET).

Reply:
i added version and vault , and made the types identical to the .Net definitions